Tools/SmokeTests/QueueDriverConnectionSmokeTest.php

```diff
@@ -39,13 +38,9 @@ public function run()
                 throw new \RuntimeException('Function isConnected() returned false.');
             }
 
-            $message = new QueueMessage();
-            $message->setTTL(1);
-            $message->setQueue('isAlive');
-
-            if (!$this->queueDriver->send($message)) {
+            if (!$this->queueDriver->send('isalive')) {
```

Comment:
```suggestion
            if (!$this->queueDriver->send('isalive', '', ['expiration' => 1000])) {
```

Reply:
This will work for AMQP, not sure for STOMP. If the header is different for STOMP, we can just add it as another key of the same array.

Reply:
I removed the "ttl" (internally is also the expieration) because what was happening was that the message after 1 second was going to the **dlq**. Don't remember with which driver this was happening. I will check in both drivers what happens

Reply:
After checking what was happening was the following. I had the **isalive** queue with the policy of the **DLX** and for this reason the messages were going to the **dlq** after 1 second. When I applied the same setup as production, we have a policy called **isalive** that is applied to the **isalive** queue and add expiration of the messages to 1 second, the messages where expiring and disappear from the queue as we expect. For this reason I removed the expiration header and it works for both drivers

I think in this case should be the user deciding what do to with this message through a policy in the broker rather than ourself defining a ttl in the code

Reply:
By definition a health check message should be short lived, either by queue policy or by message TTL. Given that framework doesn't provide any policies, nor forces you to use any (retry/throttle policies are "dumb" in framework, since they just nack the message when they are not ready to be processed), I'd say it's the smoke test function responsibility to make it short lived. If it goes to a DLX because of the users' policy, the user needs to accommodate that policy to what the framework expects. In my opinion!

Reply:
Changed

Reply:
Changed to x-message-ttl to make it compatible with our current queue policy.

Reply:
Cool!